plugins/pulp_docker/plugins/importers/sync.py

```diff
@@ -234,7 +234,10 @@ def process_main(self):
                 if content_type == man_list:
                     self._process_manifest_list(manifest, digest, available_blobs, tag)
                 else:
-                    self._process_manifest(manifest, digest, available_blobs, tag)
+                    foreign_layers = self._process_manifest(manifest, digest, available_blobs, tag)
+                    if foreign_layers:
+                        # we don't want to process schema1 manifest with foreign layers
```

Comment:
I don't recall why schema 1 is important here, vs schema 2. Can you explain?
Also, wouldn't we want to still get whatever blobs are available that are not of the foreign variety?